src/ctap/mod.rs

```diff
@@ -1028,6 +1059,49 @@ mod test {
     // ID is irrelevant, so we pass this (dummy but valid) value.
     const DUMMY_CHANNEL_ID: ChannelID = [0x12, 0x34, 0x56, 0x78];
 
+    fn check_make_response(
+        make_credential_response: Result<ResponseData, Ctap2StatusCode>,
+        flags: u8,
+        expected_aaguid: &[u8],
+        expected_credential_id_size: u8,
+        expected_extension_cbor: &[u8],
+    ) {
+        match make_credential_response.unwrap() {
+            ResponseData::AuthenticatorMakeCredential(make_credential_response) => {
+                let AuthenticatorMakeCredentialResponse {
+                    fmt,
+                    auth_data,
+                    att_stmt,
+                } = make_credential_response;
+                // The expected response is split to only assert the non-random parts.
+                assert_eq!(fmt, "packed");
+                let mut expected_auth_data = vec![
+                    0xA3, 0x79, 0xA6, 0xF6, 0xEE, 0xAF, 0xB9, 0xA5, 0x5E, 0x37, 0x8C, 0x11, 0x80,
+                    0x34, 0xE2, 0x75, 0x1E, 0x68, 0x2F, 0xAB, 0x9F, 0x2D, 0x30, 0xAB, 0x13, 0xD2,
+                    0x12, 0x55, 0x86, 0xCE, 0x19, 0x47, flags, 0x00, 0x00, 0x00,
+                ];
+                expected_auth_data.push(INITIAL_SIGNATURE_COUNTER as u8);
+                expected_auth_data.extend(expected_aaguid);
+                expected_auth_data.extend(&[0x00, expected_credential_id_size]);
+                assert_eq!(
+                    auth_data[0..expected_auth_data.len()],
+                    expected_auth_data[..]
+                );
+                /*assert_eq!(
```

Comment:
Why is this code commented out?

Reply:
Forgot to remove, deleted now. This is the random part of the returned data.